arch/arm/core/irq_manage.c

```diff
@@ -88,7 +88,7 @@ void _irq_priority_set(unsigned int irq, unsigned int prio, u32_t flags)
 	 * of priority levels reserved by the kernel.
 	 */
 
-#if CONFIG_ZERO_LATENCY_IRQS
+#ifdef CONFIG_ZERO_LATENCY_IRQS
```

Comment:
At least in the Bluetooth subsystem we prefer `#if defined(CONFIG...)` since you can extend the conditions if you want (e.g. `#if defined(CONFIG_FOO) && defined(CONFIG_BAR)`) which is something you can't do with `#ifdef`

Reply:
> which is something you can't do with #ifdef

Why, you can, just replace "#ifdef FOO" with "#if defined(FOO) && defined(BAR)" ;-). It happens rare enough, and in all other cases short "#ifdef ..." can be used.

Reply:
@pfalcon I guess it's a matter of taste - we've tried to stick to `#if` just for consistency.

Reply:
Btw, `#if IS_ENABLED(CONFIG_FOO)` would be another way, and is actually the preferred one in Linux (it's slightly different there though since there's also the "build as module" option which we don't have)

Reply:
> I guess it's a matter of taste - we've tried to stick to #if just for consistency.

True. I actually replied because your comment started with "At least in the Bluetooth subsystem we prefer ...". If there's a consistency, then it really rather be global for the whole project, not per-subsystem.

Reply:
+1 to `#if defined(xyz)`